cibuildwheel/resources/pinned_docker_images.cfg

```
@@ -1,22 +1,22 @@
 [x86_64]
-manylinux1 = quay.io/pypa/manylinux1_x86_64:2020-09-26-9f151ab
-manylinux2010 = quay.io/pypa/manylinux2010_x86_64:2020-09-26-58cbbcd
-manylinux2014 = quay.io/pypa/manylinux2014_x86_64:2020-09-26-7f7a14f
+manylinux1 = quay.io/pypa/manylinux1_x86_64:2020-10-06-c4a4a7c
+manylinux2010 = quay.io/pypa/manylinux2010_x86_64:2020-10-06-786a931
+manylinux2014 = quay.io/pypa/manylinux2014_x86_64:2020-10-06-b2ca7a1
 
 [i686]
-manylinux1 = quay.io/pypa/manylinux1_i686:2020-09-26-9f151ab
-manylinux2010 = quay.io/pypa/manylinux2010_i686:2020-09-26-58cbbcd
-manylinux2014 = quay.io/pypa/manylinux2014_i686:2020-09-26-7f7a14f
+manylinux1 = quay.io/pypa/manylinux1_i686:2020-10-06-c4a4a7c
+manylinux2010 = quay.io/pypa/manylinux2010_i686:2020-10-06-786a931
+manylinux2014 = quay.io/pypa/manylinux2014_i686:2020-10-06-b2ca7a1
 
 [pypy_x86_64]
 manylinux2010 = pypywheels/manylinux2010-pypy_x86_64:2020-09-26-8f210b5
 
 [aarch64]
-manylinux2014 = quay.io/pypa/manylinux2014_aarch64:2020-09-26-7f7a14f
+manylinux2014 = quay.io/pypa/manylinux2014_aarch64:2020-10-05-2f3b046
```

The aarch64 build failed on travis-ci fi-or the manylinux2014 aarch64 image. I restarted it and it should now be available.
```suggestion
manylinux2014 = quay.io/pypa/manylinux2014_aarch64:2020-10-06-b2ca7a1
```

---

Good catch, didn't notice that! Reran `./bin/update_dependencies.py` and indeed, this is the only change.